### PR TITLE
DMD-1387 Home Screen Header Adjustment

### PR DESCRIPTION
**Pull request by tomasz-ludek**
Fixed the issues with scaling collapsing header especially on smaller screen sizes.

Small screen:
small_screen

Medium screen:
medium_screen

Large screen:
large_screen

### CODE REVIEW

**Comment by HashEngineering** [comment]
@tomasz-ludek 
This needs to be remerged against the `redesign` branch

File: wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java [hunk 1]
```diff
@@ -62,7 +62,7 @@
 import org.bitcoinj.core.TransactionConfidence.ConfidenceType;
 import org.bitcoinj.core.listeners.DownloadProgressTracker;
 import org.bitcoinj.core.Utils;
-import org.bitcoinj.core.listeners.AbstractPeerDataEventListener;
+import org.bitcoinj.core.listeners.DownloadProgressTracker;
```
Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java]: This is a duplicate import.

Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java]: See line 63.

File: wallet/src/de/schildbach/wallet/ui/WalletActivity.java [hunk 2]
```diff
@@ -316,6 +313,13 @@ private void initNavigationDrawer() {
     protected void onResume() {
         super.onResume();
 
+        if (!initComplete) {
+            loaderManager.initLoader(ID_BLOCKCHAIN_STATE_LOADER, null, blockchainStateLoaderCallbacks);
+            initComplete = true;
+        } else {
+            loaderManager.restartLoader(ID_BLOCKCHAIN_STATE_LOADER, null, blockchainStateLoaderCallbacks);
+        }
+
```
Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: Why is this code in here?

Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: Perhaps this is a bug in GitHub's diff.

File: wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java (hunk 1 above)
Comment (tomasz-ludek) [reply wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java]: Fixed

File: wallet/src/de/schildbach/wallet/ui/WalletActivity.java (hunk 2 above)
Comment (tomasz-ludek) [reply wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: Fixed

**Pull request merged**